vortexasdk/endpoints/eia_forecasts.py

```diff
@@ -0,0 +1,80 @@
+"""EIA Forecasts Endpoint."""
+
+from vortexasdk.endpoints.eia_forecasts_result import EIAForecastResult
+from vortexasdk.endpoints.endpoints import EIA_FORECASTS_RESOURCE
+from vortexasdk.operations import Search
+from vortexasdk.api.shared_types import to_ISODate
+
+
+class EIAForecasts(Search):
+    """EIA forecasts Endpoint, use this to search through Vortexa's EIA Forecasts data.
+
+    The data includes:
+
+    - Vortexa's data science based forecast of the EIA number to be published on the week
+    - Actual EIA import/export numbers as published by the EIA Weekly Supply Estimates report
+    - EIA stocks (kbl)
+    - Cover (days of Supply for the whole of the US, as published by the EIA Weekly Supply Estimates report)
+    - Refinery runs (refiner “Percent Operable Utilization” as published by the EIA Weekly Supply Estimates report)
+
+    """
+
+    def __init__(self):
+        Search.__init__(self, EIA_FORECASTS_RESOURCE)
+
+    def search(
+        self,
+        preset: str = None,
+        filter_time_min: str = False,
```

Comment:
The type here should be datetime, rather than string. We'll also want sensible default datetimes rather than the bool `False`